### DISCUSSION TITLE
What is the name of the famous 🇻🇳 sandwich?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#800] [Galaxy Trivia] [answered]
**Bonus:** List the main ingredients and steps to make it.

**Comment by fathiputera** [comment]
The famous Vietnamese sandwich is called a "Banh Mi." Banh Mi is a flavorful and popular Vietnamese sandwich that typically consists of a crusty French baguette filled with various ingredients such as marinated meats (commonly pork or chicken), pickled vegetables, fresh herbs like cilantro, and mayonnaise or other sauces for added flavor. It is known for its combination of savory, sweet, and tangy flavors, making it a beloved street food both in Vietnam and internationally.

Ingredients:

- French baguette or Vietnamese baguette

- Meat filling (commonly pork or chicken):

  - Thinly sliced grilled or roasted pork or chicken
  
  - Marinated and grilled pork (thit nuong) or lemongrass chicken (ga nuong)

- Pickled vegetables:
  
  - Carrots, julienned or thinly sliced
  
  - Daikon radish, julienned or thinly sliced
  
  - Rice vinegar
  
  - Sugar
  
  - Salt

- Fresh vegetables and herbs:

  - Fresh cilantro (coriander) leaves
  
  - Fresh mint leaves
  
  - Fresh sliced cucumbers

- Condiments and sauces:

  - Mayonnaise
  
  - Sriracha sauce or chili sauce
  
  - Maggi sauce or soy sauce

**Comment by peterbaccaro** [comment, marked as answer]
The famous Vietnamese sandwich is called a "**Banh Mi**". Banh Mi is a delicious fusion of French and Vietnamese cuisine, combining a crispy French baguette with various fillings such as grilled meats (commonly pork or chicken), pâté, pickled vegetables, fresh herbs, and sauces like mayonnaise or chili sauce. It has gained popularity worldwide due to its unique blend of flavours and textures.

Here are the **main ingredients and steps to make a classic Banh Mi sandwich**:

**Ingredients:**

- Baguette (traditionally made with a Vietnamese baguette, which is light and crispy)
- Protein (grilled pork, grilled chicken, roasted pork belly, or tofu are common options)
- Pâté (typically pork liver pâté, but you can use other types if preferred)
- Pickled vegetables (carrots and daikon radish, thinly sliced or julienned and pickled in vinegar and sugar)
- Fresh herbs (commonly cilantro/coriander and Vietnamese mint/basil)
- Cucumber (thinly sliced)
- Chili sauce or mayonnaise (optional, but adds a tasty kick)

**Steps:**

1. Slice the baguette: Cut the baguette into sandwich-sized lengths (typically around 6-8 inches).
2. Prepare the protein: Cook the protein of your choice (grilled pork, grilled chicken, roasted pork belly, or tofu) with appropriate seasonings and marination. Ensure that the protein is cooked through and flavorful.
3. Slice the baguette open: Slice the baguette lengthwise, but not all the way through, leaving one side attached.
4. Spread pâté: If desired, spread a layer of pâté on the inside of the baguette.
5. Assemble the ingredients: Start by layering the cooked protein inside the baguette, followed by pickled vegetables, cucumber slices, and fresh herbs.
6. Add sauce: If desired, you can add chili sauce or mayonnaise for extra flavor.
7. Serve and enjoy: Close the baguette and your Banh Mi sandwich is ready to be enjoyed! It is often served as a handheld street food snack.

Feel free to customize your Banh Mi by adding other ingredients like sliced jalapeños, soy sauce, or additional vegetables to suit your taste.

**Reply by ParanoidUser** [reply]
Boom! 💥💯 You've aced it with your **century answer!** 👏🎉 Your expertise and passion are unmatched. Let's continue the excitement! 🎊🎈
